resources/server/messages/messages.service.ts

```diff
@@ -135,7 +143,7 @@ class _MessagesService {
           const participantPlayer = PlayerService.getPlayerFromIdentifier(participantIdentifier);
 
           if (participantPlayer) {
-            emitNet(MessageEvents.SEND_MESSAGE_SUCCESS, participantPlayer.source, messageData);
+            emitNet(MessageEvents.SEND_MESSAGE_SUCCESS, reqObj.source, messageData);
```

Comment:
What's going on here? We still need to send this to the 'participant'. The broadcast doesn't update any messages.